README.md

```diff
@@ -37,6 +37,13 @@ be started with:
 At which point, the application will be available at
 <http://localhost:5000>.
 
+###Static assets
```

Comment:
I think it probably makes more sense to put this above the bit about launching the server --- fits more with how you'd logically want to go about setting things up. Also I don't see any reason why this should be a third level heading as opposed to a second level? Change that third `#` to a space.